nifi-nar-bundles/nifi-standard-services/nifi-hbase_1_1_2-client-service-bundle/nifi-hbase_1_1_2-client-service/src/main/java/org/apache/nifi/hbase/HBase_1_1_2_ClientService.java

```diff
@@ -88,16 +88,16 @@
 
     static final long TICKET_RENEWAL_PERIOD = 60000;
 
-    private volatile Connection connection;
```

Comment:
I think it's best to leave these as private and expose protected getters if the values are needed.